src/status_im/contexts/wallet/account/db.cljs

```diff
@@ -0,0 +1,9 @@
+(ns status-im.contexts.wallet.account.db)
+
+(defn get-accounts
```

Comment:
What's the purpose of these functions? Why are they not `re-frame/subs` ?

Reply:
because they're used in events, not subs.

The general idea with these `db` namespaces is to have a clear layering of the functions we expose.

If I use the example of networks, you'll see:
1. The `status-im.contexts.wallet.networks.core` ns contains functions that operate on a _network_ or _collections of networks_. This is the core "API" we have for networks, and is mainly used by subs, but not limited to that.
2. The `status-im.contexts.wallet.networks.db` ns contains functions that normalize and extract networks data from the db, and some also wrap "core" functions, so instead of passing the network(s), you pass the db. They're only used by events to increase reusability and to keep events cleaner, easier to refactor and read.